apps/submission-gate/container/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:24-bookworm
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)